pkg/generators/openapi.go

```diff
@@ -242,6 +250,10 @@ func (g *openAPIGen) Init(c *generator.Context, w io.Writer) error {
 }
 
 func (g *openAPIGen) GenerateType(c *generator.Context, t *types.Type, w io.Writer) error {
+	glog.V(5).Infof("validating API rules for type %v", t)
+	if err := g.linter.validate(t); err != nil {
+		return err
```

Comment:
Errors shouldn't fail the process, otherwise we'll stop at the first one.

Reply:
The errors here are not API rule violations, but errors that cause "rules" to fail to complete. We don't have any yet but I'm imagining "rules" may hit arbitrary errors like opening file failure, HTTP connection refused, etc (probably not likely). We could aggregate those kind of errors, but as long as we return any error, `generator.Finalize` (which is where the "linter/compiler" flushes violations to report file/stdout) [won't run](https://github.com/kubernetes/gengo/blob/dcbe4570f0cf6efbc583a5321c8f9390f71a544d/generator/execute.go#L303-L310), so there is no difference from stopping at the first one. 

Another option is to remove the error and assume "rules" will always run with no error